helm/presets/local.yaml

```diff
@@ -1,3 +1,6 @@
+define:
```

Comment:
Are you also going ot need to have changes to aws.yaml?  We probably need this configured behind the ingress controller which I can help you with.  I'm okay if we do that in seperate pull requests to keep this one from getting longer.

Reply:
I will add that now. Although I'm not sure what needs to be done with the ingress controller. I'll need your help there for sure.

Reply:
Nothing needs done to the ingress _controller_ but you need to add another path to `Values.ingress.paths` in the root values.yaml file and change the define service to a ClusterIP (not required but more restricted than NodePort and obviates the need for NodePort)